Dockerfile

```diff
@@ -0,0 +1,67 @@
+# This file is part of dirtsand.
+#
+# dirtsand is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# dirtsand is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with dirtsand.  If not, see <http://www.gnu.org/licenses/>.
+
+FROM alpine
+
+COPY . /usr/src/dirtsand
+WORKDIR /usr/src
+
+ARG PRODUCT_BRANCH_ID=1
+ARG PRODUCT_BUILD_ID=918
+ARG PRODUCT_BUILD_TYPE=50
+ARG PRODUCT_UUID=ea489821-6c35-4bd0-9dae-bb17c585e680
+ARG DS_HOOD_USER_NAME=DS
+ARG DS_HOOD_INST_NAME=Neighborhood
+ARG DS_HOOD_POP_THRESHOLD=20
+
+# Everything coalesced into this single RUN statement to avoid creating WIP images.
+RUN \
+    apk add build-base cmake git openssl-dev postgresql-dev readline-dev zlib-dev && \
+    apk add bash libstdc++ openssl postgresql-client readline zlib && \
```

Comment:
```suggestion
    apk add build-base cmake git openssl-dev postgresql-dev readline-dev zlib-dev \
        bash libstdc++ openssl postgresql-client readline zlib && \
```

Reply:
Also, I'm pretty sure docker can have multiple `RUN` statements executed in sequence, rather than chaining everything into a single command...

Reply:
Right, but docker is a CoW system. Each RUN statement generates a new image. The intermediate images get rather large with build-base installed. Those could be manually discarded with docker images prune, but I'm trying to avoid recommending docker's strange cli.